.github/workflows/helm-publish.yml

```
@@ -0,0 +1,70 @@
+          cache: true
+
+      - name: Install Helm
+        uses: azure/setup-helm@v4
+        with:
+          version: 'v3.16.4'
+
+      - name: Regenerate CRDs and RBAC into the chart
+        run: make manifests
+
+      - name: Resolve chart versions from tag
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:2:30: Double quote to prevent globbing and word splitting

(shellcheck)